types/database/posts_in_channel.d.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {Relation} from '@nozbe/watermelondb';
+import Model, {Associations} from '@nozbe/watermelondb/Model';
+
+import Channel from '@typings/database/channel';
+
+/**
+ * PostsInChannel model helps us to combine adjacent posts together without leaving
+ * gaps in between for an efficient user reading experience of posts.
+ */
+export default class PostsInChannel extends Model {
```

Comment:
why is this part of this PR?

Reply:
I've removed its content.  It be added in next PR.